src/app/methods/EmailsStore.js

```diff
@@ -0,0 +1,101 @@
+import { Meteor } from 'meteor/meteor'
+import { HTTP } from 'meteor/http'
+import * as _ from 'lodash'
+
+import EmailsStore from 'app/collections/EmailsStore'
+import EmailsStoreStatus from 'app/collections/EmailsStoreStatus'
+
+
+Meteor.methods({
+  doUpdateEmailsStore: function () {
+  	var source_email_id = 'exceltes@gmail.com'
+  	var source_email_password = 'java@123'
+  	var source_host = 'imap.gmail.com'
+  	var source_port = '993'
+  	var source_encryp = 'ssl'
+
+	var last_email_id_stored = Meteor.call('getLastEmailidStored', source_email_id )
+
+	var date = new Date()
+	//--data to fetch new emails
+	var todaysDate = moment(date).format("YYYY-MM-DD")
+	var start_fetch_from_email_id = ""
+	if( last_email_id_stored != false ){
+		start_fetch_from_email_id = last_email_id_stored + 1
+	}
+
+	//-----
+	var BASE_URL = "http://excellencetechnologies.co.in/imap/?"
+	var PARAMS = ""
+	if( start_fetch_from_email_id == ''){
+		PARAMS = "email="+source_email_id+"&pass="+source_email_password+"&date="+todaysDate+"&host="+source_host+"&port="+source_port+"&encryp="+source_encryp
```

Comment:
also there in api there is limit of 100 emails only. ie it will return only 100, even if we have 2000-300 pending
so need to fire api automatically again and again from frontend